components/prospects/CreateProspect.tsx

```diff
@@ -29,6 +30,7 @@ function CreateProspect() {
           severity: 'success',
```

Comment:
I don't think we still need to set the message to `success` if we redirect the user anyway.